sql/moz-fx-data-shared-prod/contextual_services_derived/adm_forecasting_v1/metadata.yaml

```
@@ -0,0 +1,19 @@
+friendly_name: AdM Forecasting
+description: |-
+  Data for AdM forecasts: Sponsored Tiles and Suggest.
+owners:
+- skahmann@mozilla.com
+labels:
+  incremental: true
+scheduling:
+  dag_name:
+  task_name:
```

```suggestion
  dag_name: bqetl_ctxsvc_derived
```
I think this can go to `bqetl_ctxsvc_derived` DAG. cc @chelseatroy who's the [owner](https://github.com/mozilla/bigquery-etl/blob/main/dags/bqetl_ctxsvc_derived.py#L26) of this DAG to confirm if that sounds OK.

After the `dag_name: bqetl_ctxsvc_derived` is specified here we need to run `bqetl dag generate bqetl_ctxsvc_derived` to update DAG definition.

---

@chelseatroy - can you confirm this table should be added to the `bqetl_ctxsvc_derived` DAG? I'm pulling AdM usage data into a table that needs to be updated daily

---

Hi @skahmann3! Can confirm, given that it's AdM usage data!

---

Perfect, thanks Chelsea!